tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -499,37 +512,6 @@ describe('IssueList', function () {
           pathname: '/organizations/org-slug/issues/searches/789/',
         })
       );
-
-      // Need to update component
-      wrapper.setProps({
-        savedSearch: localSavedSearch,
-        location: {
-          ...router.location,
-          pathname: '/organizations/org-slug/issues/searches/789/',
-          query: {
-            sort: 'freq',
-            environment: [],
-            project: [],
-          },
-        },
-      });
-      await tick();
-      wrapper.update();
-
-      wrapper.find('IssueListSortOptions DropdownButton').simulate('click');
-      wrapper.find('DropdownItem').at(3).find('MenuItem span').at(1).simulate('click');
-
-      expect(browserHistory.push).toHaveBeenLastCalledWith(
-        expect.objectContaining({
-          pathname: '/organizations/org-slug/issues/searches/789/',
-          query: {
-            environment: [],
-            project: [],
-            sort: 'freq',
-            statsPeriod: '14d',
-          },
-        })
-      );
```

Comment:
Removing a test for the sort dropdown here since the dropdown has moved from the search & filter bar to inside issues table. I've added a new test for the same component in `actions.spec.jsx`.